aws/resource_aws_emr_cluster.go

```diff
@@ -1298,12 +1302,23 @@ func expandInstanceGroupConfigs(instanceGroupConfigs []interface{}) []*emr.Insta
 
 		applyBidPrice(config, configAttributes)
 		applyEbsConfig(configAttributes, config)
-		applyAutoScalingPolicy(configAttributes, config)
+
+		if v, ok := configAttributes["autoscaling_policy"]; ok {
+			var autoScalingPolicy *emr.AutoScalingPolicy
+
+			err := json.Unmarshal([]byte(v.(string)), &autoScalingPolicy)
+
+			if err != nil {
+				return []*emr.InstanceGroupConfig{}, fmt.Errorf("error parsing EMR Auto Scaling Policy JSON: %s", err)
```

Comment:
Surfacing this error is showing some test failures that will need to be investigated:

```
--- FAIL: TestAccAWSEMRCluster_instance_group (19.17s)
    testing.go:538: Step 0 error: Error applying: 1 error occurred:
        	* aws_emr_cluster.tf-test-cluster: 1 error occurred:
        	* aws_emr_cluster.tf-test-cluster: error parsing EMR instance groups configuration: error parsing EMR Auto Scaling Policy JSON: unexpected end of JSON input
--- FAIL: TestAccAWSEMRCluster_instance_group_EBSVolumeType_st1 (18.81s)
    testing.go:538: Step 0 error: Error applying: 1 error occurred:
        	* aws_emr_cluster.tf-test-cluster: 1 error occurred:
        	* aws_emr_cluster.tf-test-cluster: error parsing EMR instance groups configuration: error parsing EMR Auto Scaling Policy JSON: unexpected end of JSON input
```

Reply:
This has been fixed, forgot to exclude `""` 😉